src/Sulu/Bundle/WebsiteBundle/Resources/config/analytics.xml

```diff
@@ -24,16 +16,17 @@
         <service id="sulu_website.analytics.manager"
                  class="Sulu\Bundle\WebsiteBundle\Analytics\AnalyticsManager" public="true">
             <argument type="service" id="doctrine.orm.entity_manager"/>
-            <argument type="service" id="sulu_website.analytics.repository"/>
+            <argument type="service" id="sulu.repository.analytics"/>
```

Comment:
think we need to add this change to the `UPGRADE.md`

Reply:
@nnatter have added aliases for the old id and for the interface